webapp/crashstats/crashstats/models.py

```diff
@@ -57,7 +57,7 @@ def get_bugs_and_related_bugs(self, signatures):
 
 
 class BugAssociation(models.Model):
-    """Specifies assocations between bug ids in Bugzilla and signatures"""
+    """Specifies associations between bug ids in Bugzilla and signatures"""
```

Comment:
I set up pre-commit and it's now fixing typos before I can create commits.

Reply:
Does that work for all local development, or is that config specific to your local development? If the former, where was that added?

Reply:
I used it a long time ago and then decided I didn't like the way it was structured. Rehan built Therapist which I used for a while, but I think that's unmaintained now.

I noticed SRE uses pre-commit for their repositories and decided to try it out again.

https://github.com/mozilla-it/webservices-infra/blob/main/.pre-commit-config.yaml
https://github.com/mozilla/terraform-modules/blob/main/.pre-commit-config.yaml
https://github.com/mozilla-it/global-platform-admin/blob/main/.pre-commit-config.yaml

I was going to give it a little more time and then bring it up at an Observeneers meeting.

In the meantime, it's fixing spelling in PRs I'm doing in files I made a change to and I wanted to point out that's a cosmetic change.